ios/keyman/Keyman/Keyman/Classes/MainViewController/MainViewController.swift

```diff
@@ -192,7 +192,9 @@ class MainViewController: UIViewController, TextViewDelegate, UIActionSheetDeleg
     view?.addSubview(textView!)
 
     // Setup Info View
-    infoView = InfoViewController()
+    infoView = InfoViewController {
+      self.infoButtonClick(nil)
```

Comment:
This is the same function that MVC uses later for the functionality of the top-right "Done" button of the Info view.